### PR TITLE
Add onboarding ready screens

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

Changes introduce additional Ready screens. There are three variations:
1. Ready, lives on the `importer/ready` route
2. Ready with site preview, lives on the `importer/ready/preview` route
3. Not ready, lives on the `importer/ready/not` route

#### Testing instructions

Manual testing:
- go to `/start/importer/`
- enter any valid URL and hit the return key
- scanning screen will be shown with the spinner placeholder (spinner will be replaced in future improvements)
- after a few seconds, the ready screen with site preview screen will be presented

#### Screenshots
Screenshot 2021-10-22 at 18 46 28

Screenshot 2021-10-22 at 18 46 47

Related to
- [Figma design](https://www.figma.com/file/TAK5FpwTvfbENJDfHc7Aq1/Site-Importing?node-id=1930%3A21706)

### CODE REVIEW

File: client/signup/steps/import/ready/index.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+									website: convertToFrendlyWebsiteName( website ),
+									platform,
+								}
+							),
+							{ strong: createElement( 'strong' ) }
+						) }
+					</SubTitle>
+
+					<div className="import__buttons-group">
+						<NextButton>{ __( 'Import your content' ) }</NextButton>
```
Comment (a8ci18n) [inline client/signup/steps/import/ready/index.tsx:47]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Import Your Content' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-import-ready)

File: client/signup/steps/import/ready/platform-details.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+						{ createInterpolateElement( __( 'Things we <strong>can</strong> import:' ), {
+							strong: createElement( 'strong' ),
+						} ) }
+					</p>
+					<ul className={ 'import__details-list' }>
+						<li>
+							<Icon size={ 20 } icon={ check } /> { __( 'Blog posts' ) }
+						</li>
+						<li>
+							<Icon size={ 20 } icon={ check } /> { __( 'Static pages' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import/ready/platform-details.tsx:86]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'All static pages' )` *ES Score: 7*

**Review by pento — APPROVED** [review]
🎉 Screen looks good, let's get it merged. We can tackle getting the functionality of it working in a follow up.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6906984

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6906984).

**Pull request merged**